modules/ngx_http_upstream_session_sticky_module/ngx_http_upstream_session_sticky_module.c

```diff
@@ -246,16 +234,8 @@ ngx_http_upstream_session_sticky_init_peer(ngx_http_request_t *r,
     sspd->sscf = sscf;
     sspd->get_rr_peer = ngx_http_upstream_get_round_robin_peer;
 
-    r->upstream->peer.data = sspd;
```

Comment:
We assert that (sspd == &sspd->rr).
(refer to nginx core module: ngx_http_upstream_ip_hash_module.c)